Hw1Task2/Hw1Task2/BWT.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+Console.WriteLine("Input string: ");
+string str = Console.ReadLine();
+string transformedString = Transformation.BWT(str);
+Console.WriteLine(transformedString);
+Console.WriteLine(Transformation.BWTinverse(transformedString));
+
+// class with two public methods for burrows-wheeler transformation
+static class Transformation
+{
+    public static string BWT(string str)
+    {
+        str += '$';
+        string[] suffixes = GetSuffixes(str);
+        sort(suffixes);
+        string result = "";
+        for (int i = 0; i < suffixes.Length; i++)
+        {
+            int index = str.Length - suffixes[i].Length - 1;
+            result += str[index < 0 ? str.Length - 1 : index];
+        }
+        return result;
+    }
+
+    private static string[] GetSuffixes(string str)
+    {
+        string[] suffixes = new string[str.Length];
+        for (int i = 0; i < str.Length; i++)
+        {
+            suffixes[i] = str.Substring(i);
+        }
+        return suffixes;
+    }
+
+    private static void sort(string[] suffixes)
```

Comment:
```suggestion
    private static void Sort(string[] suffixes)
```
Методы всегда именуются с заглавной